docusaurus/docs/iOS/components/message-composer.md

```diff
@@ -69,41 +66,39 @@ class CustomMessageListVC: UIViewController {
     override open func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
 
-        // Setup the keyboard observer
-        keyboardObserver.register()
+        keyboardHandler.start()
     }
 
     override open func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
 
-        // Remove the keyboard observer
-        keyboardObserver.unregister()
+        keyboardHandler.stop()
     }
 }
 ```
-As you can see if you want to use the `ComposerVC` in your own message list view you need to setup the dependencies of the composer, add it as a child view controller of your custom message list view controller and even manage the keyboard yourself or use our keyboard observer to manage it.
+As you can see if you want to use the `ComposerVC` in your custom message list view you need to setup the dependencies of the composer, add it as a child view controller of your custom message list view controller, and even manage the keyboard yourself or use our keyboard observer to manage it.
 
 ### Customization
 
-The `ComposerVC` and `ComposerView` are completely customizable. You can not only change the UI layout and styling, but you can extend the composer functionality as well. In case you want to change the styling, adding new views and new functionality you can take a look at the [Customize Message Composer](../guides/customize-message-composer) guide. If you want to add a new custom attachment and make the composer to support it, you should read the [Message Composer Custom Attachments](../guides/working-with-custom-attachments) guide. 
+The `ComposerVC` and `ComposerView` are completely customizable. You can not only change the UI layout and styling, but you can extend the composer functionality as well. In case you want to change the styling, adding new views, and new functionality you can take a look at the [Customize Message Composer](../guides/customize-message-composer) guide. If you want to add a new custom attachment and make the composer support it, you should read the [Message Composer Custom Attachments](../guides/working-with-custom-attachments) guide.
```

Comment:
```suggestion
The `ComposerVC` and `ComposerView` are highly customizable in both styling and functionality directions. In case you want to change the styling, adding new views, and new functionality you can take a look at the [Customize Message Composer](../guides/customize-message-composer) guide. If you think about introducing new custom attachment and make the composer support it, read the [Message Composer Custom Attachments](../guides/working-with-custom-attachments) guide.
```